File: TextIndexApi/src/main/java/com/hideakin/textsearch/index/model/FileGroupRequest.java
package com.hideakin.textsearch.index.model;

public class FileGroupRequest {

	private String name;

	public FileGroupRequest() {
		this(null);
	}

	public FileGroupRequest(String name) {
		this.name = name;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: TextSearchCLI/model/FileGroupRequest.cs
using Newtonsoft.Json;

namespace com.hideakin.textsearch.model
{
    internal class FileGroupRequest
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        public FileGroupRequest(string name)
        {
            Name = name;
        }
    }
}
